news/2023/2023-05-01-the-osu-legacy-of-reesewashere.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "The osu! YouTube legacy of ReeseWasHere"
+date: 2023-05-01 10:00:00 +0000
+---
+
+Before livestreaming and player YouTube channels were widespread, you could rely only on one man with a camera...
+
+![](/wiki/shared/news/2023-05-01-osu-legacy-of-reesewashere/banner.jpg)
+
+If you were even slightly engaged with the osu! community between 2015 and 2016, you probably heard of [ReeseWasHere](https://www.youtube.com/@ReeseWasHere), seen some of his YouTube videos, or at the very least seen the saying "Reese get the camera!!" whenever a top player would set a nice score. If you ever saw the ["mom get the camera"](https://www.youtube.com/watch?v=mLRbZJS5A_E) meme from the Call of Duty community, Reese would be the osu! equivalent to that. So without further ado, let's get into the history of this legendary community member and see what they are up to now!
+
+Reese originally found osu! by watching a League of Legends streamer called [BoxBox](https://www.twitch.tv/boxbox), who used to play osu! while waiting in queue for another game. He liked the fact you can play anime songs, and even without any rhythm game experience he gave it a try. After playing for a fair bit and dabbling in mapping, he realized that what he really loved doing is spectating top players, watching their streams on Twitch and sharing epic moments with the osu! community. In his own words:
+
+> I really wanted to focus on liveplays because I wanted to capture them in the moment. I wanted it to be where you could relive the moment it happened just by watching the video, so I always incorporated Twitch chat or the ingame chat when I could. And even for those that weren't there live you could still feel like you was there by seeing people's reactions.
+
+As if by divine intervention, the initiative to share amazing moments perfectly coincided with [Cookiezi](https://osu.ppy.sh/users/124493/osu)'s (now known as chocomint) unban after being banned for 2 years, and he was sure to capture that:
```

Comment:
```suggestion
As if by divine intervention, the initiative to share amazing moments perfectly coincided with [Cookiezi](https://osu.ppy.sh/users/124493/osu)'s (now known as chocomint) return to the game after being banned for 2 years, and he was among the first to capture the moment:
```

Reply:
I figured saying unban could be an issue, but it was more factual than just the "return" seeing as they were playing for a few months before getting the unban, I would compromise by saying "return to online play"

Reply:
```suggestion
As if by divine intervention, the initiative to share amazing moments perfectly coincided with [Cookiezi](https://osu.ppy.sh/users/124493/osu)'s (now known as chocomint) return to the online play after being banned for 2 years, and he was among the first to capture the moment:
```